source/NVDAObjects/window/winword.py

```diff
@@ -424,6 +451,11 @@ def activate(self):
 		mathMl=mathPres.getMathMlFromTextInfo(self)
 		if mathMl:
 			return mathPres.interactWithMathMl(mathMl)
+		newRng=self._rangeObj

```

Comment:
I think you mean newRng=self._rangeObj.duplicate? otherwise you are mutating the TextInfo itself.